web/templates/personal_menu.hbs

```diff
@@ -73,9 +71,7 @@
                     </a>
                 </li>
                 {{/if}}
-            </ul>
-
-            <ul class="list">
+                <div class="navbar-dropdown-divider"></div>
```

Comment:
Replacing these grouping `<ul>` elements with incorrect `<div>` elements that are dividers seems like a big step backwards.

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/div.20inside.20ul.20.5Bresolved.5D